machines/Issuers/IssuersActions.ts

```diff
@@ -43,6 +44,21 @@ export const IssuersActions = (model: any) => {
           isVerified: false,
         }),
     }),
+    setIsExpired: assign({
+      vcMetadata: (context: any, event: any) => {
+        return new VCMetadata({
+          ...context.vcMetadata,
+          isExpired: event.data.verificationErrorCode == EXPIRED_VC_ERROR_CODE,
+        });
+      },
+    }),
+    resetIsExpired: assign({
+      vcMetadata: (context: any, event: any) =>
+        new VCMetadata({
+          ...context.vcMetadata,
+          isExpired: false,
+        }),
+    }),
```

Comment:
no need to have separate action for isVerifed and isExpired.
Have a action to handle verification result.